packages/ek-components/src/components/EkCardButtons.vue

```diff
@@ -127,6 +139,14 @@ export default {
 <style lang="scss" scoped>
 @import '../styles.scss';
 
+.card-buttons {
+  &::after {
+    content: '';
+    display: block;
+    clear: both;
+  }
+}
```

Comment:
Nit: this change has nothing to do with the commit message.

Reply:
Oops, that should be in its own _issue_, even: Joana and I noticed the margins were inconsistent when we were looking at this. Good thing to keep track of properly :) I'll move that change elsewhere for orderliness and then merge this. Thank you for the review!

Reply:
I split this into https://github.com/endlessm/kolibri-explore-plugin/issues/690.